setup.py

```diff
@@ -11,7 +11,7 @@
 
 requirements = [
     # Until COMPAS reaches 1.0, we pin major.minor and allow patch version updates
-    'compas>=0.16.2,<0.17',
+    'compas>=0.16.3,<0.17',
```

Comment:
I think this is necessary because of the change of location of `xtransform` and `xform_from_transformation`.

Reply:
Shouldn't we bump to `0.16.4`?

Reply:
Can do.